docs/Cloud_integrations/Device_Integration_Center/overview.md

```diff
@@ -0,0 +1,88 @@
+# Data Integration Center
+
+Data Integration Center is our feature that allows you to forward device related data to your systems, databases,
+message brokers etc. There are two kind of events:
+
+* Lifecycle - these are related to device life cycle events like creation, update, deletion
+* Telemetry - these are related to telemetry gathered by device like temperature measurement
+
+![overview](./images/data-integration-center-overview.webp)
+
+## Templating
+
+Templating feature allow you to define how your payload (or headers) looks like. This enables you to integrate with
+almost all available services, gathering your data where you desire.
+
+![wizard-form-body](./images/wizard-form-body.webp){ width=50% }
+
+You are free to modify body template as long as it proper JSON. If you want different content type you can use `Text`
+that have not validation at all.
+
+```json
+{
+  "version": "$version",
+  "timestamp": $timestamp,
+  "endpointName": "$endpointName",
+  "domainId": "$domainId",
+  "url": "$url",
+  "value": "$value"
+}
+```
+
+As you can notice there are values starting with `$` e.g. `$endpointName`. These are called variables.
+
+## Variables
+
+Variables are predefined, and allows you to use some device context in payload. When sending variables are replaced with
```

Comment:
Variables are predefined and allow for using some device context in a payload. When sending variables are replaced with actual value e.g. `$endpointName` with `test-device-47193512895`. There are several variables that can be used, if variable does not exist e.g. `$notExisting` then it is sent as it is. Available variables differs from context where they are used.